README.md

```diff
@@ -30,6 +30,7 @@ This module makes the following assumptions:
 - `allow_major_version_upgrade` - Allow upgrading of major version of database (eg. from Postgres 9.5.x to Postgres 9.6.x), default: false
 - `auto_minor_version_upgrade` - Automatically upgrade minor version of the DB (eg. from Postgres 9.5.3 to Postgres 9.5.4), default: true
 - `apply_immediately` - Specifies whether any database modifications are applied immediately, or during the next maintenance window, default: false
+- `maintenance_window` - The window to perform maintenance in. Syntax: 'ddd:hh24:mi-ddd:hh24:mi' UTC, default: "Mon:00:00-Mon:03:00"
```

Comment:
Default value does not match - https://github.com/terraform-community-modules/tf_aws_rds/pull/26/files#diff-c9ac8098c5ea9d3e6a9a596ff0c512a4R71 . The rest looks good.